pkg/addons/default/testdata/sample-1.11.json

```diff
@@ -534,7 +534,7 @@
                     }
                   }
                 ],
-                "image": "602401143452.dkr.ecr.eu-west-1.amazonaws.com/amazon-k8s-cni:v1.4.1",
```

Comment:
Don't need to update these manifests, they should come from live clusters, and don't really have to be up-to-date, these are just samples.